internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1433,7 +1433,7 @@ func (r *ClusterDeploymentsReconciler) populateLogsURL(ctx context.Context, log
 }
 
 func (r *ClusterDeploymentsReconciler) eventsURL(log logrus.FieldLogger, clusterId string) (string, error) {
```

Comment:
```suggestion
func (r *ClusterDeploymentsReconciler) generateEventsURL(log logrus.FieldLogger, clusterId string) (string, error) {
```

Reply:
Done in https://github.com/openshift/assisted-service/pull/3111